constants.js

```diff
@@ -24,5 +41,6 @@ const COUNTRIES = {
 
 module.exports = {
   SKUS: SKUS,
+  SKUS_AUSTRALIA: SKUS_AUSTRALIA,
```

Comment:
nice - probably could remove the comment on line 39 that notes issues with AU stock

Reply:
Done :)